internal/config/secrets.go

```diff
@@ -35,13 +35,13 @@ func loadSecrets(config *load.Config) {
 		if secret.Kind == "" {
 			load.Logrus.WithFields(logrus.Fields{
 				"secret": name,
-			}).Error("config: secret is missing kind")
+			}).Error("config: secret missing: 'kind'")
```

Comment:
this makes it look like a secret named **kind** missing. I would keep the other phrase or write something like:
`config:  secret 'kind' is required` or `config:  secret 'kind' is missing`  or like you have below :
`config: secret needs 'kind' parameter to be set`